include/itkPolarToCartesianTransform.h

```diff
@@ -111,24 +111,29 @@ class ITK_TEMPLATE_EXPORT PolarToCartesianTransform:
    */
   OutputPointType TransformPoint(const InputPointType  &point ) const ITK_OVERRIDE;
 
-  /** Method to transform a vector - not applicable for this type of
-      transform. */
+  /** Method to transform a vector - not applicable for this type of transform. */
   OutputVectorType TransformVector(const InputVectorType &) const ITK_OVERRIDE
     {
     itkExceptionMacro(<< "Method not implemented yet.");
     return OutputVectorType();
     }
 
-  /** Method to transform a vnl_vector - not applicable for this type of
-      transform. */
+  /** Method to transform a vnl_vector - not applicable for this type of transform. */
   OutputVnlVectorType TransformVector(const InputVnlVectorType &) const ITK_OVERRIDE
     {
     itkExceptionMacro(<< "Method not implemented yet.");
     return OutputVnlVectorType();
     }
 
-  /** Method to transform a CovariantVector - not applicable for this type of
-      transform */
+  /** Method to transform a vector - not applicable for this type of transform. */
+  OutputVectorPixelType TransformVector(
+      const InputVectorPixelType &, const InputPointType &) const ITK_OVERRIDE
+    {
+    itkExceptionMacro(<< "Method not implemented yet.");

```

Comment:
Is this method "not applicable"  (comment )or is it "not implemented yet" (exception)?

Reply:
Not applicable. The "not implemented" was quick copy-paste. I am in the process of fixing the Linux compile error too.